### PR TITLE
Fix long display name overflowing reply tile on IRC layout

### PR DESCRIPTION
**Pull request by luixxiul**
This PR fixes the issue that a long display name overflows `ReplyTile` on IRC layout.

It also adds a stress test to check that long strings do not overflow and are displayed with ellipsis where needed on each layout.

Fixes https://github.com/vector-im/element-web/issues/24738

|Before|After|
|---------|------|
|Screenshot from 2023-03-05 04-26-39|Screenshot from 2023-03-05 04-26-50|

type: defect

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix long display name overflowing reply tile on IRC layout ([\#10273](https://github.com/matrix-org/matrix-react-sdk/pull/10273)). Fixes vector-im/element-web#24738. Contributed by @luixxiul.

### CODE REVIEW

**Comment by weeman1337** [comment]
Hello @luixxiul thank you for your contribution. Does this relate to any issue? If yes, can you link it?

Can you always add screenshots from before and after for visual changes?

**Comment by luixxiul** [comment]
Thanks for the heads-up. I'm going to do them later, before making this PR ready for review.

File: res/css/views/rooms/_IRCLayout.pcss [hunk 1]
```diff
@@ -185,9 +185,10 @@ $irc-line-height: $font-18px;
     .mx_ReplyChain {
         margin: 0;
         .mx_DisambiguatedProfile {
-            order: unset;
             width: unset;
             background: transparent;
+            order: unset;
+            flex-shrink: unset; /* Unset flex-shrink to prevent long display name blowout */
```
Comment (luixxiul) [inline res/css/views/rooms/_IRCLayout.pcss:191]: This resets `flex-shrink: 0;` specified above.

**Comment by luixxiul** [comment]
Removed the branch by mistake. Recreating a PR.
